lib/milestone-db/index.js

```diff
@@ -23,17 +25,23 @@ MilestoneDB.prototype.close = function(callback) {
  *   the signature (error, snapshot) => void;
  */
 MilestoneDB.prototype.getMilestoneSnapshot = function (collection, id, version, callback) {
-  process.nextTick(callback, null, undefined);
+  var error = new ShareDBError(5019, 'getMilestoneSnapshot MilestoneDB method unimplemented');
+  if (callback) return process.nextTick(callback, error);
+  this.emit('error', error);
 };
 
 /**
  * @param {string} collection - name of the snapshot's collection
  * @param {Snapshot} snapshot - the milestone snapshot to save
  * @param {Function} callback (optional) - a callback to invoke after the snapshot has been saved.
- *   Should have the signature (error, wasSaved) => void;
+ *   Should have the signature (error) => void;
  */
 MilestoneDB.prototype.saveMilestoneSnapshot = function (collection, snapshot, callback) {
-  var saved = false;
-  if (callback) return process.nextTick(callback, null, saved);
-  this.emit('save', saved, collection, snapshot);
+  var error = new ShareDBError(5020, 'saveMilestoneSnapshot MilestoneDB method unimplemented');
+  if (callback) return process.nextTick(callback, error);
+  this.emit('error', error);
+};
+
+MilestoneDB.prototype._isValidVersion = function (version) {
```

Comment:
Optional request:

Now that this method is just a wrapper, does it still need to exist (like for testing purposes or subclass overrides), or can it be removed?